### ISSUE TITLE
PROCEDURA DI ATTUAZIONE: VERIFICA INDIRIZZO EMAIL (UTENTE ANONIMO)

### ISSUE THREAD

**Issue by eborzacchi** [AgID/agid-forms-frontend#62]
## Comportamento atteso
Accesso alla Procedura di attuazione: http://form.agid.gov.it-131.1.241.146.nip.io/form/procedura-attuazione/bcdec6ad-e8e0-4834-ab22-73b4327e4345/
Cliccando sul pulsante "verifica l'indirizzo email", viene verificato il codice ricevuto per email e se corretto, il form mi visualizza l'indirizzo email che ho inserito e posso procedere con la compilazione del form.

## Comportamento attuale
A) Inserendo il codice ricevuto via email, nell'apposito campo del form, cliccando sul pulsante "verifica l'indirizzo email": DAL FORM SCOMPARE IL CAMPO "Indirizzo email (o PEC)" con l'indirizzo che avevo inserito (vedi immagini allegate: "DCD_Vedifica email_1" e "DCD_Vedifica email_2"
B) Facendo il refresh della pagina, l'indirizzo email "elenaborzacchi.almaviva@gmail.com" che avevo inserito, mi appare in alto nel form. 
![DCD_Vedifica email_1](https://user-images.githubusercontent.com/58256267/73839407-355d2a80-4816-11ea-8e07-b42c53e3fc29.png)
![DCD_Vedifica email_2](https://user-images.githubusercontent.com/58256267/73839411-368e5780-4816-11ea-9143-46dd8513d54f.png)
![DCD_Vedifica email_3](https://user-images.githubusercontent.com/58256267/73839413-368e5780-4816-11ea-97f0-54468ecc00f8.png)

a destra nel campo riservato all'utente autenticato (vedi immagine allegata: "DCD_Vedifica email_3")

## Contesto
TEST SUI SEGUENTI BROWSER: 
A) Google Chrome V. 79.0.3945.130 (Build ufficiale) (a 64 bit)
B) Internet Explorer V. 11.864.17763.0
C) Firefox V. 72.0.2 (64 bit)

**Comment by pdavide** [comment]
Quello che segnali è il comportamento atteso e non è un bug.
Se lo ritieni opportuno possiamo continuare a mostrare l'indirizzo email anche dopo che è stato verificato.

**Comment by eborzacchi** [comment]
No, se questo era il requisito ok, è che in genere l'utente si aspetta di vedere tutto quello che ha inserito.

**Comment by pdavide** [comment]
Io non ero presente in fase di definizione dei requisiti, quindi non saprei dirti.
Se per te non era chiaro questo comportamento vale la pena di modificarlo lasciando la visualizzazione dell'indirizzo email.

**Comment by eborzacchi** [comment]
A mio avviso l'indirizzo email inserito dall'utente dovrebbe rimanergli visibile.
Invece l'anomalia (segnalato nel punto B sopra) dell'indirizzo email che compare in alto a destra?

**Comment by pdavide** [comment]
Anche quella del punto B non è un'anomalia ma il comportamento atteso dell'applicazione dato che la verifica dell'indirizzo email "genera" un login nell'applicazione.
Anche in questo caso possiamo modificare il comportamento e non visualizzare l'indirizzo email in alto anche se io procederei per lasciare questo comportamento dato che segnala all'utente il suo stato "autenticato/mail-verificate" rispetto all'applicazione.

**Comment by eborzacchi** [comment]
Non ti so rispondere, sentiamo Claudio cosa dice su questa cosa. 
L'ho segnalata perché l'avevamo già riscontrata come anomalia nel precedente test, Bug Fixing su Drive https://drive.google.com/file/d/1WL8wDydHSEp1x7cHU8HzrMgpm4lTi-uR/view?usp=sharing, nella Sezione "INVIA" feedback -> A) CASO 1
